controllers/backup.go

```diff
@@ -49,27 +49,30 @@ var (
 
 // returns then name of the last backup resource, or,
 // if this is the first time to run the backup or current backup is not found, a newly generated name
-func (r *BackupReconciler) getActiveBackupName(backup *v1beta1.Backup, c client.Client, ctx context.Context) string {
+func (r *BackupReconciler) getActiveBackupName(
+	ctx context.Context,
+	backup *v1beta1.Backup,
+	c client.Client,
+) string {
 
 	if backup.Status.CurrentBackup == "" || isBackupPhaseFinished(backup.Status.Phase) {
 
 		// no active backup, return newyly generated backup name
 		return getVeleroBackupName(backup.Name, backup.Spec.VeleroConfig.Namespace)
-	} else {
-		// check if current backup resource exists
+	}
 
-		veleroIdentity := types.NamespacedName{
-			Namespace: backup.Spec.VeleroConfig.Namespace,
-			Name:      backup.Status.CurrentBackup,
-		}
-		// get the velero CR using the veleroIdentity
-		veleroBackup := &veleroapi.Backup{}
-		err := r.Get(ctx, veleroIdentity, veleroBackup)
-		if err != nil {
-			// unable to get backup resource, create a new one
-			backup.Status.CurrentBackup = ""
-			return getVeleroBackupName(backup.Name, backup.Spec.VeleroConfig.Namespace)
-		}
+	// check if current backup resource exists
+	veleroIdentity := types.NamespacedName{
+		Namespace: backup.Spec.VeleroConfig.Namespace,
+		Name:      backup.Status.CurrentBackup,
+	}
+	// get the velero CR using the veleroIdentity
+	veleroBackup := &veleroapi.Backup{}
+	err := r.Get(ctx, veleroIdentity, veleroBackup)
+	if err != nil {
```

Comment:
you can create new one if the velero backup doesn't exists... But you should check the `err` type. If it's a `NotFound` but you may have other errors (for example `APISever` cannot answer`... Unsure If I understand the logic correctly but at least the comment is misleading

Reply:
This is just formatting change to reduce line length and remove useless else block, no changes in the logic here, so I would keep it as is in this PR :)